lib/drivers/medtronic600/NGPHistoryParser.js

```diff
@@ -1017,6 +1023,19 @@ class NGPHistoryParser {
     return null;
   }
 
+  // This is here rather than in the simulator because the boluses both share a bolusNumber
+  // as an identifier.
+  findMatchingBolusProgrammedEvent(bolus) {
+    const matchingBolusIndex = _.findIndex(this.events, event => event === bolus);
+
+    return _.findLast(
+      this.events, event => event.eventType === bolus.eventType - 199 &&
```

Comment:
What is the 199 - can it be a constant definition instead?

Reply:
I'll make a const (and document it)